File: WEEK3/DAY1/Assignment/Quesn1/Quesn1.cpp
//Reverse an integer array
#include <bits/stdc++.h>

using namespace std;

void reverseArray(int P[],int start,int end){

    while (start < end)
    {
        int temp = P[start];
        P[start] = P[end];
        P[end] = temp;
        start++;
        end--;
    }
    
}

void printArray(int P[],int n){
    for(int i = 0;i<n;i++){
        cout<<P[i]<<" ";
    }
}

int main(){
    int P[] = {5,8,7,6};
    int n = sizeof(P)/sizeof(P[0]);
    reverseArray(P,0,n-1);
    printArray(P,n);
    return 0;
}

File: WEEK3/DAY1/Assignment/Quesn1/Quesn1.java
//Reverse an integer array
public class Quesn1 {
    static void reverseArray(int P[],int start,int end){
        int temp;
        while(start < end){
            temp = P[start];
            P[start] = P[end];
            P[end] = temp;
            start++;
            end--;
        }
    }
    static void printArray(int P[],int n){
        for(int i = 0;i<n;i++){
            System.out.print(P[i] + " ");
        }
    }
    
    public static void main(String args[]){
        int P[] = {5,8,7,6};
        int n = P.length;
        reverseArray(P,0,n-1);
        printArray(P,n);
    }
}